pkgs/servers/guacamole-client/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, nixosTests
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "guacamole-client";
+  version = "1.5.2";
+
+  src = fetchurl {
+    url = "https://archive.apache.org/dist/guacamole/${finalAttrs.version}/binary/guacamole-${finalAttrs.version}.war";
+    hash = "sha256-hu/DABbkA4lI2MGlw3oLBeONrdMQTbbsA3VbxuMRHEA=";
+  };
+
+  dontUnpack = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/webapps
+    cp $src $out/webapps/guacamole.war
```

```suggestion
    cp $src $out/share/guacamole/guacamole.war
```

or is that a tomcat thing?

---

Yes,

So we can do:

```nix
      webapps = [
        cfg.clientPackage
      ];
```